Dockerfile

```diff
@@ -0,0 +1,62 @@
+# syntax = docker/dockerfile:1
```

Comment:
Since I wasn't using docker for this project, I haven't updated this to ensure that it works. But, if I wanted to expand on this, I'd rather have the basic scaffold ready for me.